server/src/service/player-bind/player-bind.service.ts

```diff
@@ -41,11 +41,11 @@ export class PlayersBindService {
             this.loggerService.errorLog(error);
             throw new Error(error);
         } else {
-            this.userRepository.addUser(playersBind.players[0]);
-            this.userRepository.addUser(playersBind.players[1]);
+            await this.userRepository.addUser(playersBind.players[0]);
+            await this.userRepository.addUser(playersBind.players[1]);
             this.loggerService.infoLog('Room and players have been bound');
 
-            return true;
+            return Promise.resolve(true);
```

Comment:
why promise.resolve, you can just return true, async method wil wrap this value into Promise automatically